haystack/preprocessor/preprocessor.py

```diff
@@ -23,6 +23,7 @@ def __init__(
         split_length: int = 1000,
         split_overlap: int = 0,
         split_respect_sentence_boundary: bool = True,
+        language: str = "english",
```

Comment:
In other parts of Haystack, we use the two-letter ISO codes ("en", "de", "es"...), see https://github.com/deepset-ai/haystack/blob/master/haystack/file_converter/pdf.py#L19

I think it would be better to stick with them and use them consistently across haystack.
I am pretty sure there's an easy conversion option from one format to the other (e.g. lookup table or small helper lib).

Reply:
That sounds reasonable.
There are a few helper libs around ([this](https://pypi.org/project/iso-639/)) just to mention one, but adding an external dependency just for this sounds overkill to me, do you agree?

Since we're supporting a limited number of languages (those from nltk) I guess a simple look-up dictionary would do.
What do you think?

Reply:
Yes, absolutely. That's why I tried to understand how many languages are supported by nltk :)
With the list you shared, a lookup dict send reasonable. Probably in `haystack/utile.py`

Reply:
Done, but import `haystack.utils` gave me a circular import.

Reply:
hey @tholor, any updates on this?

Reply:
Did you resolve the circular import issue or shall I have a look here?

Reply:
hey @tholor, no I didn't resolve it because I'm not sure about the best approach to solve it, and it looked "too much" for just that iso_639 dictionary.

The reason is `haystack.utils` imports `haystack.document_store.sql`, that imports `haystack.document_store.base`, that imports `haystack.preprocessor.preprocessor`. So when if we import `haystack.utils` inside `haystack.preprocessor.preprocessor` we start the loop all over again.

If you have any idea on the best approach to solve this, I could fix it on a different PR :)

Reply:
Ok got it. I also think it's not worth the effort right now. If there's additional util methods coming up that we need in preprocessor we can  investigate it further